app/views/admin/tenants/index.html.erb

```diff
@@ -1,21 +1,72 @@
 <div id="tenants_header">
   <h1>Tenants</h1>
 </div>
+<div id="unapproved">
+  <h2>Unapproved Tenants</h2>
+  <table style="width:100%">
+    <tr>
+      <th>Organization</th>
+      <th>Location</th>
+      <th>Slug</th>
+    </tr>
 
-<table style="width:100%">
-  <tr>
-    <th>Organization</th>
-    <th>Location</th>
-    <th>Slug</th>
-  </tr>
-
-  <% @tenants.each do |tenant| %>
+    <% if @unapproved %>
+      <% @unapproved.each do |tenant| %>
+        <tr>
+          <td><%= tenant.organization %></td>
+          <td><%= tenant.location %></td>
+          <td><%= tenant.slug %></td>
+          <td><%= link_to "EDIT", edit_admin_tenant_path(tenant.id) %></td>
```

Comment:
Eventually, it would be nice to have this EDIT button replaced with separate "Approve" "Take Offline" "Go Live" as appropriate. This PR is already big, so I'm opening #177 to address this.